### ISSUE TITLE
TieredMenu: Items are not read by NVDA on hover

### ISSUE THREAD

**Issue by ppc-harmoney** [primefaces/primeng#15187]
### Describe the bug

We are using prime ng in an application that is legally required to be WCAG compliant. To check compliance we are using [Axe devtools](https://www.deque.com/axe/browser-extensions/#:~:text=Overview-,Overview,your%20time%20with%20false%20positives.) and [NVDA screenreader](https://www.nvaccess.org/download/). 

It appears that the menu items are read out loud by the NVDA screenreading software only when focusing them with the keyboard, not when hovering over them with the mouse. 

![image](https://github.com/primefaces/primeng/assets/84312273/76fcf1b9-91f0-4278-b8ff-30ee65b2965b)

### Environment

Windows 11 Pro

### Angular version

17.3.1

### PrimeNG version

17.12.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.2

### Browser(s)

chrome 123

### Steps to reproduce the behavior

Start NVDA and open a menu (in this case of a splitbutton). You will hear the menu items being read when using the keyboard, but not when using the mouse.

### Expected behavior

The menu item should be read out loud in both cases.

**Comment by cetincakiroglu** [comment]
Hi,

Thanks a lot for reporting the issue!
